unittests/test_schedulers.py

```diff
@@ -781,30 +757,13 @@ def test_flex_alloc_invalid_constraint(make_flexible_job):
         prepare_job(job)
 
 
-def test_flex_alloc_valid_reservation_cmd(make_flexible_job):
-    job = make_flexible_job('all',
-                            sched_access=['--constraint=f2'],
-                            sched_reservation='dummy')
```

Comment:
This test should remain except that the reservation should be passed through `sched_options`.